libindy/src/api/anoncreds.rs

```diff
@@ -525,19 +597,22 @@ pub extern fn indy_prover_create_and_store_claim_req(command_handle: i32,
 pub extern fn indy_prover_store_claim(command_handle: i32,
                                       wallet_handle: i32,
                                       claims_json: *const c_char,
-                                      rev_reg_json: *const c_char,
+                                      rev_reg_def_json: *const c_char,
+                                      rev_reg_entry_json: *const c_char,
```

Comment:
What does rev_reg_entry_json mean?

Reply:
It is transaction that contains accumulator value